tests/fixtures/inversify-dynamic-container/server.ts

```diff
@@ -0,0 +1,19 @@
+import * as bodyParser from 'body-parser';
+import * as express from 'express';
+import * as methodOverride from 'method-override';
+
+import './managedController';
+import { RegisterRoutes } from './routes';
+
+export const app: express.Express = express();
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use(methodOverride());
+RegisterRoutes(app);
+
+// It's important that this come after the main routes are registered
+app.use((err: any, req: express.Request, res: express.Response, next: express.NextFunction) => {
+  res.status(err.status || 500).send(err.message || 'An error occurred during the request.');
+});
+
+app.listen(3004);
```

Comment:
```suggestion
app.listen();
```

Reply:
Ah! Would `app.listen(3010);`, or something, be better?

Reply:
I think hardcoding ports is always a bad idea here, but the other servers do it aswell, so I can't fault you for setting one (one that's not used by another server).
If you omit the port, an available one will be picked.

Reply:
Honestly, I copied the fixture from the 'inversify' fixture. 😅. I'll accept the suggestion if the port can be omitted.